ZooProject/src/zoo/Personen/Personal.java

```diff
@@ -2,50 +2,78 @@
 
 import zoo.Zoo;
 import zoo.Gehege.Gehege;
-//import zoo.Tier.Tier;		//frag mich ob personal das benutzen müsst also wegen ITier & lebtin
-import zoo.Tier.ITier;
+import zoo.Tier.Tier;		//frag mich ob personal das benutzen mÃ¼sst also wegen ITier & lebtin
 
 public class Personal extends Personen {
 
-	private Zoo arbeitsplatz; // Name kann nur wieder ausgegeben werden
-	private int betreut_gehege_id;
+	private Zoo arbeitsplatz;
+	private Gehege[] gehege;
```

Comment:
UML sagt das Personen nur ein Gehege in ihrem Typ haben dürfen dswg hab ich es über das betreut_gehege_id gemacht ich würde es eher so drin lassen.
Kann man aber auch so lassen, weil ich selbst keine Lust mehr habe.